azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/SetupFragment.kt

```diff
@@ -97,6 +102,22 @@ internal class SetupFragment :
     }
 
     private fun setActionBarTitle() {
-        activity?.title = ""
+        val mSpannableText = SpannableString(getString(R.string.azure_communication_ui_call_setup))
+        var mTitleColor = Color.BLACK
```

Comment:
Can these colors be part of the Theme (or theme attributes) and not hard coded?

Reply:
I.e. this link might help
https://stackoverflow.com/questions/5861661/actionbar-text-color

Can be applied in our default light/dark theme files?